instagramapp/src/App.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar'
+import PostContainer from './components/PostContainer/PostContainer'
+
+
+
+// The root App component of your application should import the dummy data from the dummy-data.js file with import dummyData from './dummy-data'; and iterate over said data, passing each individual object as a prop to an instance of PostContainer.
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      likes: dummyData.likes
+    };
+  };
+
+  //In `app.js` use `componentDidMount()` to set your data to the component's state. 
+
+  //post-render actions (fetching data or event listeners)
+  componentDidMount() {
+    console.log('CDM Running')
+    //mock data fetch
+    setTimeout(() => {
+      this.setState({
+        data: dummyData
+      });
+    }, 2000);
+
+    window.addEventListener('resize', () => {});
+  }
+
+  //clean up event listeners/subscriptions
+  componentWillUnmount() {
+    window.removeEventListener('resize', () => {});
+  }
+
+  incrementLike = () => {
+    let likes = this.state.likes + 1;
+    this.setState({ likes });
```

Comment:
youve got this in your app.js, it would probably make more sense to put your likes in state of your posts.js since there is a likes variable for each post.